src/components/publishing/icon/expand.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from "react"
+
+class IconExpand extends Component<any, null> {
+  render() {
+    return (
+      <svg viewBox="0 0 27 27" version="1.1" xmlns="http://www.w3.org/2000/svg">
+        <g stroke="none" strokeWidth="1" fill="none" fillRule="evenodd">
+          <g>
+            <circle fill="#000000" fillRule="nonzero" opacity="0.6" cx="13.9575195" cy="13.0588379" r="13" />
+            <g transform="translate(8.000000, 7.000000)" stroke="#FFFFFF">
+              <g>
+                <g transform="translate(0.382532, 0.382532)">
+                  <polyline points="4.78165571 0 11.1571966 0 11.1571966 6.37554094" />
+                  <polyline
+                    transform="translate(3.187769, 7.969426) rotate(-180.000000) translate(-3.187769, -7.969426) "
+                    points="-1.01881144e-06 4.78165537 6.37553992 4.78165537 6.37553992 11.1571963"
+                  />
+                  <path d="M10.360254,0.796942618 L0.796942618,10.360254" strokeLinecap="square" />
+                </g>
+              </g>
+            </g>
+          </g>
+        </g>
+      </svg>
```

Comment:
How do you make these SVG icons? Or does design output those for you?

Reply:
Ah design outputs these for us. They're generated via Sketch

Reply:
Dope!